setup.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+import codecs
+from setuptools import setup, find_packages
+
+
+PACKAGE_NAME = "allure2html"
+CMD = "allure2html"
+
+
+def read(fname):
+    file_path = os.path.join(os.path.dirname(__file__), fname)
+    return codecs.open(file_path, encoding='utf-8').read()
+
+
+setup(
+    name=PACKAGE_NAME,
+    version='1.0.0',
+    author='Sine.wang',
+    author_email='sinecelia.wang@gmail.com',
```

Comment:
If you don't mind, I'd ask you to add my username and email to the "author" and "author_email" fields space-separated with yours.

Reply:
Hi, @MihanEntalpo 
Yes, of course, I don't mind.  I will do that later.